ebean-test/src/test/java/org/tests/json/TestJsonMapBasic.java

```diff
@@ -50,6 +59,36 @@ public void whereManyPredicate() {
     assertThat(query2.getGeneratedSql()).contains("select distinct on (t0.id) ");
   }
 
+  @Test
+  @ForPlatform(Platform.DB2)
+  public void whereManyPredicateDb2() {
```

Comment:
Unfortunately, the test is not yet starting on current master branch, because of the  `maxTableNameLength` and `maxConstraintNameLength` of 18 char limits in DB2-platform. (changing them to 128 and the test will run)
We will introduce a Legacy/DB2LUW/... platform as suggested  in #2484 soon